packages/botonic-plugin-text-classification/src/utils/locale-utils.ts

```diff
@@ -0,0 +1,11 @@
+import { Locale } from '@botonic/nlp/lib/types'
+import franc from 'franc'
+import langs from 'langs'
+
+export function detectLocale(input: string, locales: Locale[]): Locale {
```

Comment:
I think it should be done in another PR because we need to also move the `locale-utils` from `plugin-ner`. 
I've opened a [ticket](https://linear.app/hubtype/issue/BOT-351/move-locale-utils-from-nlp-plugins).